.github/workflows/build.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5
         with:
-          go-version: 1.20.x
+          go-version: 1.21.x
```

Comment:
so our OSP have support for 1.21.x right ?

Reply:
I think so, but it doesn't matter here. `catalog-cd` is not a productized tool, and is not shipped in OSP either.